### ISSUE TITLE
Wrap line in message editor

### ISSUE THREAD

**Issue by ModProg** [Xithrius/twitch-tui#262]
**Is your feature request related to a problem? Please describe.**
When writing long messages, you cannot see the beginning.

**Describe the solution you'd like**
Wordwrap the input box, increasing its vertical size when writing more.

**Comment by Xithrius** [comment]
Perhaps there should be a config option for this, perhaps some users would want to see more chat while they're typing.

Thoughts?

**Comment by ModProg** [comment]
Makes sense to have an option for this. But probably you wont be typing messages long enough to fill the entire screen. Especially with the length limit.

**Comment by Xithrius** [comment]
👌
